spec/requests/api/chargebacks_spec.rb

```diff
@@ -0,0 +1,85 @@
+require "spec_helper"
+
+RSpec.describe "chargebacks API" do
+  include Rack::Test::Methods
+
+  def app
+    Vmdb::Application
+  end
+
+  before { init_api_spec_env }
+
+  it "can fetch the list of all chargeback rates" do
+    chargeback_rate = FactoryGirl.create(:chargeback_rate)
+
+    api_basic_authorize
+    run_get chargebacks_url
+
+    expect_result_resources_to_include_data(
+      "resources",
+      "href" => [
+        "http://example.org/api/chargebacks/#{chargeback_rate.to_param}"
+      ]
+    )
+    expect_result_to_match_hash(@result, "count" => 1)
+    expect_request_success
+  end
+
+  it "can show an individual chargeback rate" do
+    chargeback_rate = FactoryGirl.create(:chargeback_rate)
+
+    api_basic_authorize
+    run_get chargebacks_url(chargeback_rate.id)
+
+    expect_result_to_match_hash(
+      @result,
+      "description" => chargeback_rate.description,
+      "guid"        => chargeback_rate.guid,
+      "id"          => chargeback_rate.id,
+      "href"        => "http://example.org/api/chargebacks/#{chargeback_rate.to_param}"
```

Comment:
no need for full path here, you can just have "href" => chargebacks_url(chargeback_rate.to_param)